packages/trip-details/src/TripDetails.story.js

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { withA11y } from "@storybook/addon-a11y";
+import { withInfo } from "@storybook/addon-info";
+import styled from "styled-components";
```

Comment:
Suggestion: Move `styled` import below React's.

Reply:
I'm declining to do this since it would no longer be an alphabetic organization of imports.